src/main/java/seedu/address/ui/PersonProfile.java

```diff
@@ -0,0 +1,57 @@
+package seedu.address.ui;
+
+import java.util.Comparator;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+import javafx.scene.layout.FlowPane;
+import javafx.scene.layout.HBox;
+import javafx.scene.layout.Region;
+import seedu.address.model.person.Person;
+
+/**
+ * An UI component representing the profile with details of a {@code Person}.
+ */
+public class PersonProfile extends UiPart<Region> {
+    private static final String FXML = "PersonProfile.fxml";
+
+    /**
+     * Note: Certain keywords such as "location" and "resources" are reserved keywords in JavaFX.
+     * As a consequence, UI elements' variable names cannot be set to such keywords
+     * or an exception will be thrown by JavaFX during runtime.
+     *
+     * @see <a href="https://github.com/se-edu/addressbook-level4/issues/336">The issue on AddressBook level 4</a>
+     */
+
+    public final Person person;
+
+    @javafx.fxml.FXML
+    private HBox cardPane;
+    @FXML
+    private Label name;
+    @FXML
+    private Label id;
+    @FXML
+    private Label phone;
+    @FXML
+    private Label address;
+    @FXML
+    private Label email;
+    @FXML
+    private FlowPane tags;
+
+    /**
+     * Creates a {@code PersonCode} with the given {@code Person} and index to display.
+     */
+    public PersonProfile(Person person) {
+        super(FXML);
+        this.person = person;
+        name.setText(person.getName().fullName);
+        phone.setText(person.getPhone().value);
+        address.setText(person.getAddress().value);
+        email.setText(person.getEmail().value);
+        person.getTags().stream()
```

Comment:
Currently throws error for me.

Steps to reproduce:
1. Start application.
2. Write `view 2`, press enter.

Result:
- `person.toString` printed to UI, as expected
- PersonProfile does not appear; unexpected

Call chain:
- `handleCommandEntered` @ CommandBox:45
- (lambda)
- `executeCommand` @ `MainWindow`:207
- `handleView` @ `MainWindow`:176
- `<init>` @ `PersonProfile`:53

Reply:
The issue was with the data representation, since the person's tag was not properly being displayed in the PersonProfile.fxml implementation in this commit. 

The later commits properly display a person's tags, although further Ui styling must be done soon.